test/unit/test_assembly.py

```diff
@@ -15,6 +15,7 @@
 import tempfile
 import argparse
 import itertools
+import gzip
```

Comment:
I think this import can be avoided... `test_assembly.py` already imports util.file which provides `open_or_gzopen` which is the more common pattern for opening gzip files.